### PR TITLE
Update doc on resources

### LINKED ISSUE #3526: Push options: resources.init
### Where on docs.platform.sh should be changed?

https://docs.platform.sh/environments.html#push-options
https://docs.upsun.com/environments.html#push-options

### What exactly should be updated?

Git 24 introduced a new push-option `resources.init` to define which strategy to use on push, possible strategies are:

- `manual`: fail the deployment and ask the user to configure the resources manually.
- `minimum`: initialize new apps and services with the minimum required resources.
- `default`: initialize new apps and services with a default amount of resources.
- `parent`: initialize new apps and services using the parent environment sizes with a fallback to default.

### Additional context

_No response_

### LINKED ISSUE #3527: Source integrations: environment_init_resources option
### Where on docs.platform.sh should be changed?

TBD

### What exactly should be updated?

TBD

### Additional context

_No response_

### LINKED ISSUE #3532: Resources: minimum and defaults
### What needs to be documented?

TBD

### Additional context

See requirements in Context field.

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

Closes #3532 
Closes #3526
Closes #3527

## What's changed

- [x] New structure for Manage Resources section as it was becoming more and more comprehensive
- [x] Add minimum resources to docs (#3532)
- [x] Document strategies without a source integration (#3526)
- [x] Document strategies with a source integration (#3526)
- [x] Add units to tables
- [x] Add links to pricing page
- [x] Update environment sync section

### CODE REVIEW

File: sites/friday/src/manage-resources/resource-init.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+For example, if you [set up a GitHub integration](), use the following options:
+
+```bash
+platform integration:add \
+  --project {{< variable "PROJECT_ID" >}} \
+  --type github \
+  --repository {{< variable "OWNER/REPOSITORY" >}} \
+  --token {{< variable "GITHUB_ACCESS_TOKEN" >}} \
+  --base-url {{< variable "GITHUB_URL" >}}
+  --resources-init= {{< variable "INITIALIZATION_STRATEGY" >}}
```
Comment (nicogommen) [inline sites/friday/src/manage-resources/resource-init.md:106]: @AnouckColson, I would remove the `=` here, to use the same pattern as the other options above.

Comment (AnouckColson) [reply sites/friday/src/manage-resources/resource-init.md:106]: Ah yes, that's a bad copy-paste on my part

Comment (AnouckColson) [reply sites/friday/src/manage-resources/resource-init.md]: Fixed!

**Review by chadwcarlson — APPROVED** [review]

File: sites/friday/src/manage-resources/resource-init.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  - "scaling"
+---
+
+When you first deploy your {{% vendor/name %}} project, and whenever you add a new app or service,
+{{% vendor/name %}} allocates the following default resources to every container:
+
+| Resource type               | Amount |
+| --------------------------- | ----------- |
+| CPU                         | 0.5 |
+| RAM                         | Depends on the [container profile](#advanced-container-profiles). |
```
Comment (nicogommen) [inline sites/friday/src/manage-resources/resource-init.md:22]: I think the link here should be `/manage-resources/adjust-resources.html#advanced-container-profiles` (or maybe to the `.md` extension, not sure how it works)

File: sites/friday/src/manage-resources/resource-init.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+| Disk size (only applicable if the app or service requires a disk)                   | 512 MB |
+
+If you don't want to use those default resources, you can define a different [resource initialization strategy](#resource-initialization-strategies).
```
Comment (nicogommen) [inline sites/friday/src/manage-resources/resource-init.md:25]: I think the link here should be `#define-a-resource-initialization-strategy`

File: sites/friday/src/manage-resources/resource-init.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+You can also [adjust those resources](#configure-resources) after your project or new container has been deployed.
```
Comment (nicogommen) [inline sites/friday/src/manage-resources/resource-init.md:26]: I think the link here should be `/manage-resources/adjust-resources.html` (or maybe to the `.md` extension, not sure how it works)

File: sites/friday/src/manage-resources/adjust-resources.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+which causes a short downtime.
+
+<--->
+
++++
+title= From the Console
++++
+
+1. Navigate to your environment.
+2. Select an app or service in the tree on the left-hand side:
```
Comment (nicogommen) [inline sites/friday/src/manage-resources/adjust-resources.md:66]: We added some new buttons to open the `Configure resources` page, that are a bit more easily accessible.

You can now open the `Configure resources` page from the environment card (or project card) directly:
Screenshot 2023-12-05 at 5 15 41 PM

Or from the `Apps & Services` graph too, but without having to open a specific app or service:
Screenshot 2023-12-05 at 5 15 52 PM

I think both of these buttons offer an easier path than the one we describe currently.
Could we open the doc accordingly?
Let me know if you need a cleaner screenshot.

File: sites/friday/src/manage-resources/adjust-resources.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+  If you've made no other changes, this redeployment causes no downtime.
+
+<--->
+
++++
+title= From the Console
++++
+
+1. Navigate to your environment.
+2. Select an app or service in the tree on the left-hand side:
```
Comment (nicogommen) [inline sites/friday/src/manage-resources/adjust-resources.md:144]: Same comment as the previous one about the new buttons to open the `Configure resources` page.

File: sites/friday/src/manage-resources/resource-init.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+## Define a resource initialization strategy
+
+You can define which strategy {{% vendor/name %}} uses to allocate resources
+when you first deploy your project or add a new container.
+The following strategies are available:
+
+| Strategy | Description |
+| ---------| ----------- |
+| `default`  | Initializes the new containers using the [{{% vendor/name %}} default resources](/manage-resources/resource-init.md).</br>This is the strategy that applies when you first deploy your project or new container, unless you explicitly set another strategy.  |
+| `manual`   | With this strategy, the first deployment fails and you need to configure resources manually through [the Console](#adjust-your-project-resources), or using `resources:set` in the CLI. |
```
Comment (AnouckColson) [inline sites/friday/src/manage-resources/resource-init.md:40]:
```suggestion
| `manual`   | With this strategy, the first deployment fails and you need to configure resources manually through [the Console](/manage-resources/adjust-resources.md), or using `resources:set` in the CLI. |
```

File: sites/friday/src/manage-resources/resource-init.md (hunk 2 above)
Comment (AnouckColson) [reply sites/friday/src/manage-resources/resource-init.md]: Fixed

File: sites/friday/src/manage-resources/resource-init.md (hunk 3 above)
Comment (AnouckColson) [reply sites/friday/src/manage-resources/resource-init.md]: Fixed

File: sites/friday/src/manage-resources/resource-init.md (hunk 4 above)
Comment (AnouckColson) [reply sites/friday/src/manage-resources/resource-init.md]: Fixed

**Pull request merged**